src/AndroidClient/android/src/androidTest/java/net/servicestack/android/TestServiceTestsAsync.java

```diff
@@ -187,7 +187,7 @@ public void complete() {
 
         ResponseStatus status = thrownError[0].getResponseStatus();
 
-        assertEquals("not here", status.getErrorCode());
+        assertEquals("NotFound", status.getErrorCode());
```

Comment:
Yeah this was a recent change to use the StatusCode as the ErrorCode instead of re-using the ErrorMessage.